api/test/run_test.sh

```diff
@@ -18,8 +18,7 @@ succeeded=$?
 if [[ $succeeded -eq 0 ]]; then
 	printf "${green}:-) All tests passed${endcol}\n"
 else
-    printf "Full system logs:\n"
-    docker-compose logs
-	printf "${red}:'( Some tests did not pass${endcol}\n"
+    printf "\n\n${red}:'( Some tests did not pass${endcol}\n"
```

Comment:
This bit seems to be using 4 spaces for indentation instead of one tab as used in the rest of the file.

Reply:
The file was already mixed. I've fixed it to use four spaces everywhere.